### PR TITLE
Add Goreport badge

### PR DESCRIPTION
**Pull request by asifdxtreme**

### CODE REVIEW

**Comment by surajnarwade** [comment]
@asifdxtreme , LGTM :+1: , it will be awesome if you send corrections for goreport resullts

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Kompose (Kubernetes + Compose)
 
-[![Build Status Widget]][Build Status] [![Coverage Status Widget]][Coverage Status] [![GoDoc Widget]][GoDoc] [![Slack Widget]][Slack]
+[![Build Status Widget]][Build Status] [![Coverage Status Widget]][Coverage Status] [![GoDoc Widget]][GoDoc] [![Slack Widget]][Slack]  [![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes/kompose)](https://goreportcard.com/report/github.com/kubernetes/kompose)
```
Comment (cdrage) [inline README.md]: Hi, please add the URL to the bottom of the page, you'll see where the other URL's are.

**Comment by asifdxtreme** [comment]
@cdrage Thanks for the review, I have updated the changes as per the suggestion

**Comment by asifdxtreme** [comment]
@surajnarwade Thanks for the suggestion, I will send the correction for goreport in later PR's.

File: README.md (hunk 1 above)
Comment (asifdxtreme) [inline README.md]: Moved to the URL's to the bottom of the page.

**Comment by asifdxtreme** [comment]
@cdrage the PR is updated as per your suggestions, can you please help me to merge this PR

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Kompose (Kubernetes + Compose)
 
-[![Build Status Widget]][Build Status] [![Coverage Status Widget]][Coverage Status] [![GoDoc Widget]][GoDoc] [![Slack Widget]][Slack]
+[![Build Status Widget]][Build Status] [![Coverage Status Widget]][Coverage Status] [![GoDoc Widget]][GoDoc] [![Slack Widget]][Slack]  [![GoReportCard Widget]][GoReportCardResult]
```
Comment (cdrage) [inline README.md]: One last nitpick (sorry!), mind moving slack to being the last widget / move goreportcard forward? Other than that, LGTM after this has been changed!

Comment (asifdxtreme) [inline README.md]: yes that makes sense, thanks for bringing this up, changed as per the suggestion

**Comment by asifdxtreme** [comment]
@cdrage Thanks for the suggestion, I have done the changes as requested

**Comment by cdrage** [comment]
I appreciated it! thanks man.

**Pull request merged**